plugins/modules/pagerduty_alert.py

```diff
@@ -5,17 +5,19 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from __future__ import absolute_import, division, print_function
+
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+DOCUMENTATION = """
```

Comment:
Please note that extensive reformatting reduces the chance of someone reviewing this, since it is a lot of extra work to work through all changes and determine which actually do change semantics and which are purely formatting.

Reply:
Cool, will remove the purely formatting changes. I autoformatted when saving ...